### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

File: locales/nl.po [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+"Een toepassing voor het converteren en optimaliseren van jpeg-, png en"
+" webp-afbeeldingen (gebaseerd op YOGA)"
+
+#: yoga_image_optimizer/about_dialog.py:38
+msgid "translator-credits"
+msgstr "Heimen Stoffels <vistausss@fastmail.com>"
+
+#: yoga_image_optimizer/image_formats.py:48
+msgid "WebP (lossless)"
+msgstr "Webp (zonder kwaliteitsverlies)"
```
Comment (flozz) [inline locales/nl.po:184]: The authors of the format use an upercse for the P of WebP, I think we should keep it as the lossy version of WebP is not internationalized and have the upercase P :)

```suggestion
msgstr "WebP (zonder kwaliteitsverlies)"
```

**Review by flozz — CHANGES_REQUESTED** [review]
Thank you for your help with the YOGA Image Optimizer translation!

Can you review the two strings I annotated then I will be able to merge your PR :)

File: locales/nl.po (hunk 1 above)
Comment (Vistaus) [reply locales/nl.po:184]: In Dutch, we usually write it like this, but I understand your reasoning, so I'll commit your suggestion. :)

File: locales/nl.po [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+#: yoga_image_optimizer/data/ui/settings-window.glade:208
+#: yoga_image_optimizer/data/ui/main-window.glade:215
+msgid "Next to the input file (→ image.opti.png)"
+msgstr "Naast het invoerbestand (→ afbeelding.opti.png)"
+
+#. Do not translate the 'optimized/' part of the example file path, it is not localized
+#: yoga_image_optimizer/data/ui/settings-window.glade:224
+#: yoga_image_optimizer/data/ui/main-window.glade:229
+msgid "In a subfolder (→ optimized/image.png)"
+msgstr "In een onderliggende map (→ geoptimaliseerd/afbeelding.png)"
```
[the remark this answers is not in the document]
Comment (Vistaus) [reply locales/nl.po]: My bad - didn't know that! Thanks for the head-up. :)

**Review by flozz — APPROVED** [review]

**Pull request merged**